android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -401,6 +401,7 @@ public void onDismiss()
         } );
     }
 
+    @Deprecated
```

Comment:
I don't think this is necessary right, since `return sdkConfiguration.getConsentDialogState().ordinal();` already triggers a linter warning. Also this isn't applied to iOS either.

Reply:
Right, will remove this.